x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/connectors/create_connector/components/choose_connector_selectable.tsx

```diff
@@ -0,0 +1,226 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { useState } from 'react';
+
+import {
+  EuiBadge,
+  EuiIcon,
+  EuiInputPopover,
+  EuiSelectable,
+  EuiSelectableOption,
+} from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+
+import dropbox from '../assets/dropbox.svg';
+
+interface ChooseConnectorSelectableProps {
+  connectorSelected: string;
+  setConnectorSelected: Function;
+}
+interface OptionData {
+  secondaryContent?: string;
+}
+
+export const ChooseConnectorSelectable: React.FC<ChooseConnectorSelectableProps> = ({
+  setConnectorSelected,
+  connectorSelected,
+}) => {
+  const connectorsData = [
```

Comment:
We have this data that you can use in stack similar to this.  https://github.com/elastic/kibana/blob/main/x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/connectors/select_connector/select_connector.tsx#L82

Reply:
Will it be available in Serverless as well? We need to get the connectors data in both either Stack or ES3

Reply:
No, it uses a different way to get the data. I think it is fair to make it work in stack or serverless first and move it to the other later.